dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/event/ProgramMessageController.java

```diff
@@ -67,7 +67,7 @@
 @RestController
 @RequestMapping(value = "/messages")
 @ApiVersion({DhisApiVersion.DEFAULT, DhisApiVersion.ALL})
-public class ProgramMessageController extends AbstractCrudController<ProgramMessage> {
+public class ProgramMessageController extends AbstractFullReadOnlyController<ProgramMessage> {
```

Comment:
Does this remove some endpoints? If this was a bug I think this change should be highlighted and not hidden in a `docs` PR/issue 😅

Reply:
this docs PR and is not touching any other bug.
used `AbstractFullReadOnlyController` instead of `AbstractCrudController` in order to avoid generation of unnecessary PUT and POST mapping `paths` as we dont need it in case of ProgramMessage